tree/tree/test/TOffsetGeneration.cxx

```diff
@@ -83,12 +83,74 @@ class TOffsetGeneration : public ::testing::Test {
       }
       file->Write();
 
-      delete random;
       delete tree2;
       delete file;
+
+      file = new TFile("TOffsetGeneration5.root", "RECREATE");
```

Comment:
Same for this commit. Please fuse with the first commit where the test has been added (and reorder if needed).

Reply:
Done.